### PR TITLE
Add ReeseUnityDemos, ubump, and note about UPM monorepos

### PR DESCRIPTION
**Pull request by reeseschultz**

### CODE REVIEW

**Comment by reeseschultz** [comment]
FYI, [ReeseUnityDemos](https://github.com/reeseschultz/ReeseUnityDemos) is a UPM monorepo with multiple packages in its `Packages/` directory, including DOTS-based navigation, a spawning package, and a randomization package. They can be imported into Unity, standalone, since each package has its own subtree branch. They all have READMEs with installation instructions. Soon they will have SemVer tags, which is why I built [ubump](https://github.com/reeseschultz/ubump), which automates all the bumping.

**Comment by starikcetin** [comment]
I'll take a look when I get the chance.

File: README.md [hunk 1]
```diff
@@ -121,7 +123,7 @@ An awesome list of Git repositories for Unity that support Unity Package Manager
 
 # How to support UPM in my Git repository?
 
-UPM wants a package.json file at the root. There are ways of doing this without changing your whole repository structure. Take a look at [this amazing article](https://www.patreon.com/posts/25070968) by mob-sakai.
+UPM requires a package.json file at the root, or, alternatively for a UPM monorepo, one package.json file per subdirectory (typically with each of those directories inside `/Packages`). There are ways of doing this without changing your whole repository structure. Take a look at [this amazing article](https://www.patreon.com/posts/25070968) by mob-sakai.
```
Comment (starikcetin) [inline README.md]: While this is a great addition, I think it confuses the reader a little bit, especially if they have no prior experience with upm.

Even if you have a monorepo, the resulting branches still have their corresponding `package.json` files at the root when you are publish for upm.

This is exactly what I tried to explain in the next sentence with `There are ways of doing this without changing your whole repository structure.`, which applies wonderfully for a monorepo.

That being said, I agree that this explanation of mine is a little mean on the verbosity side. It needs a good rewording.

What do you think?

Comment (reeseschultz) [reply README.md]: I don't know what other people find or do not find confusing, so I'm not prepared to reword README further. Now, I would be happy to change your wording back to what it was so you can decide how to reword it later if you want. Alternatively, if you want me to commit new wording, let me know what it is and I'd be happy to do that too.

Comment (starikcetin) [reply README.md]: Let's merge this as is now and rewrite that section completely afterwards.

**Comment by starikcetin** [comment]
Thank you! @reeseschultz

**Pull request merged**